src/metabase/xrays/domain_entities/specs.clj

```diff
@@ -59,7 +59,7 @@
   (partial m/map-kv-vals (fn [k v]
                            (assoc v :name k))))
 
-(def ^:private LegacyMetrics
+(def ^:private Metrics
```

Comment:
I believe this is actually just referring to the "Metrics" in the template files so "LegacyMetrics" was an incorrect change I made a while back.